lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -3018,7 +3019,8 @@ public function setOriginalEntityProperty($oid, $property, $value)
      */
     public function getEntityIdentifier($entity)
     {
-        return $this->entityIdentifiers[spl_object_hash($entity)];
+        return $this->entityIdentifiers[spl_object_hash($entity)]
```

Comment:
Can you please create a test that will cover the exception part?

Reply:
Meaning what?  A test that will cause the Exception to be thrown?

Reply:
Yes. By making `getEntityIdentifier($entity)` fail with an entity that isn't in `$this->entityIdentifiers`

Reply:
ping @oojacoboo

Reply:
@smilesrg I've added a test.  Trying to run `phpunit` locally failed with `Interface 'Doctrine\Persistence\NotifyPropertyChanged' not found`.  Sounds like a dependency to another repo not included as a vendor lib.

Reply:
I can't run unit tests locally at all: phpunit reports `No tests executed!`

Reply:
@oojacoboo try to rebase onto origin, then run `composer update` and then execute tests again